Plugins/SafeDIGenerator/SafeDIGenerateDependencyTree.swift

```diff
@@ -39,16 +59,32 @@ struct SafeDIGenerateDependencyTree: BuildToolPlugin {
                 outputSwiftFile.path(),
             ]
 
-            let toolPath: URL = if FileManager.default.fileExists(atPath: Self.armMacBrewInstallLocation) {
-                // SafeDITool has been installed via homebrew on an ARM Mac.
-                URL(filePath: Self.armMacBrewInstallLocation)
-            } else if FileManager.default.fileExists(atPath: Self.intelMacBrewInstallLocation) {
-                // SafeDITool has been installed via homebrew on an Intel Mac.
-                URL(filePath: Self.intelMacBrewInstallLocation)
+            let downloadedToolLocation = context.downloadedToolLocation
+            if context.hasSafeDIFolder, downloadedToolLocation == nil {
+                Diagnostics.error("""
+                \(context.safediFolder.path()) exists, but there is no SafeDITool binary for this release present.
+
+                To download the current release SafeDITool binary, run:
+                \tswift package --package-path \(context.package.directoryURL.path()) --allow-network-connections all --allow-writing-to-package-directory safedi-release-install
+
+                To use a debug SafeDITool binary instead, remove the `.safedi` directory by running:
+                \trm -rf \(context.safediFolder.path())
+                """)
```

Comment:
what the error looks like in practice:
<img width="803" alt="IMG_9199" src="https://github.com/user-attachments/assets/4e0b87f7-2393-4ad5-92d9-0f5eb5ef6216">

Reply:
If there is a `.safedi/` directory at the package root, I'm taking that to mean that the user intended to be using a release version of `SafeDITool`. Users will likely enter this state after installing `SafeDITool`, then bumping the `SafeDI` version they are using.